config/routes.rb

```diff
@@ -11,9 +11,14 @@
     post "/login", to: "sessions#create"
     delete "/logout", to: "sessions#destroy"
 
-    resources :users
+    resources :users do
+      member do
```

Comment:
so sánh member và collection giúp a

Reply:
member route sử dụng cho mỗi 1 thành phần của resource, nó sẽ tạo route với tham số truyền vào là id của thành phần đấy
collection route sử dụng chung cho resource, nó sẽ tạo route chung cho cả resource mmaf không cần truyền id. Nó hoạt động trên toàn bộ resource